docs/UserGuide.md

```diff
@@ -223,19 +263,36 @@ Examples:
 *  `list` followed by `edit 1 p/12345678 animal/Bob` edits the phone number and the pet name of the 1st fosterer in the address book to 12345678 and Bob respectively.
 *  `list` followed by `edit 2` changes the view to the profile page of the 2nd fosterer in the address book since parameters are not provided.
 
-Expected Output (success):
+Expected output (if successful):
+Format 1: 
+```agsl
+Edited Fosterer: [NAME]; Phone: [NUMBER]; Email: [EMAIL]; Address: [ADDRESS]; Housing: [HOUSING]; Availability: [AVAILABILITY]; Animal name: [ANIMAL NAME]; Animal type: [ANIMAL TYPE]; Tags: [TAG]
 ```
-Foster family details successfully edited!
+*  If a field is not edited, the output shows the fosterer’s original data of the field.
+
+Format 2: 
+```agsl
+Viewing Fosterer: [NAME]; Phone: [NUMBER]; Email: [EMAIL]; Address: [ADDRESS]; Housing: [HOUSING]; Availability: [AVAILABILITY]; Animal name: [ANIMAL NAME]; Animal type: [ANIMAL TYPE]; Tags: [TAG]
 ```
+*  The UI generates a pop-up window where details of the fosterer are shown.
+*  Aside from the details added by the add command, here is the list of fields the pop-up window shows:
+    *  NOTE : Notes for the user, examples include but are not limited to (optional and non-exhaustive):
+        *  Health condition of the animal
+        *  Duration / end date of foster, whichever the foster manager prefers
+        *  Identifiable physical traits of the animal
+      
+<div markdown="span" class="alert alert-primary">
+  :bulb: <b>Tip:</b> <code>NAME</code>, <code>NUMBER</code>, <code>EMAIL</code>, and <code>ADDRESS</code> are compulsory. 
+Empty compulsory values disable the save command that saves the edited fosterer data.
+</div>
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:** Edit may cause information loss. 
+Before you type the save command, make sure there are no accidental overwrites with faulty information. 
+</div>
 
 Expected Output (failure):
-1. Compulsory fields are not filled in
-```
-Oops! Compulsory fields are not filled in!
-```
-2. System Error
-```
-Oops! There seems to be an error, please check your fields again.
+```agsl
+Oops! There seems to be an error, please check the format of your input again.
```

Comment:
If you are planning to change output to this, then can just leave it like this i guess